### PR TITLE
Lotus2 tests

### PR DESCRIPTION
**Pull request by hildebra**
just added optional to a few labels to make this clearer, remjust added optional to a few labels to make this clearer, removed blast, vserach, usearch optionsoved blast, vserach, usearch options

### CODE REVIEW

File: tools/lotus2/lotus2.xml [hunk 1]
```diff
@@ -213,10 +213,10 @@ exit \$EXIT_VALUE
...
-        <param argument="-forwardPrimer" type="text" value="" label="Forward primer used to amplify DNA region" help="E.g. 16S primer fwd" />
-        <param argument="-reversePrimer" type="text" value="" label="Reverse primer used to amplify DNA region" help="E.g. 16S primer rev" />
+        <param argument="-forwardPrimer" type="text" value="" label="Forward primer used to amplify DNA region (optional)" help="E.g. 16S primer fwd" />
+        <param argument="-reversePrimer" type="text" value="" label="Reverse primer used to amplify DNA region (optional)" help="E.g. 16S primer rev" />
         <conditional name="offtarget_cond">
-            <param argument="-offtargetDB" type="select" label="Remove likely contaminant OTUs/ASVs based on alignment to host genome" help="Useful for low-bacterial biomass samples to remove possible host genome contaminations">
+            <param argument="-offtargetDB" type="select" label="Remove likely contaminant OTUs/ASVs based on alignment to host genome (optional)" help="Useful for low-bacterial biomass samples to remove possible host genome contaminations">
```
Comment (nsoranzo) [inline tools/lotus2/lotus2.xml]:
```suggestion
            <param argument="-offtargetDB" type="select" label="Remove likely contaminant OTUs/ASVs based on alignment to host genome" help="Useful for low-bacterial biomass samples to remove possible host genome contaminations">
```

File: tools/lotus2/lotus2.xml [hunk 2]
```diff
@@ -271,13 +271,10 @@ exit \$EXIT_VALUE
...
                 <param argument="-taxAligner" type="select" label="Taxonomy aligner for taxonomic profiling of OTUs">
                     <option value="0" selected="true">RDPclassifier (max likelihood)</option>
-                    <option value="1">Blast LCA against custom reference database</option>
                     <option value="2">LAMBDA LCA against custom reference database</option>
-                    <option value="3">UTAX likelihood corrected</option>
-                    <option value="4">VSEARCH LCA against custom reference database</option>
                 </param>
                 <when value="0">
-                    <param argument="-rdp_thr" type="float" min="0" max="1" value="0.8" label="Confidence threshold for RDP"/>
+                    <param argument="-rdp_thr" type="float" min="0" max="1" value="0.8" label="Confidence threshold for RDP (optional)"/>
```
Comment (nsoranzo) [inline tools/lotus2/lotus2.xml]: This is presently not optional in the Galaxy wrapper (if you remove the default value, Galaxy gives you an error when you click "Run"). But I can make it optional, if you like.

File: tools/lotus2/lotus2.xml [hunk 3]
```diff
@@ -286,7 +283,7 @@ exit \$EXIT_VALUE
                     <expand macro="refDB_macro" />
                 </when>
                 <when value="3">
-                    <param argument="-utax_thr" type="float" min="0" max="1" value="0.8" label="Confidence threshold for UTAX"/>
+                    <param argument="-utax_thr" type="float" min="0" max="1" value="0.8" label="Confidence threshold for UTAX (optional)"/>
```
Comment (nsoranzo) [inline tools/lotus2/lotus2.xml]: Same as for `-rdp_thr`

File: tools/lotus2/lotus2.xml (hunk 2 above)
Comment (hildebra) [reply tools/lotus2/lotus2.xml]: yes please make this optional (like basically most variables in lotus2, there are good defaults programmed in if the flag is not being set)

**Review by nsoranzo — APPROVED** [review]

**Pull request merged**